querydsl-apt/src/main/java/com/querydsl/apt/DefaultConfiguration.java

```diff
@@ -195,6 +196,8 @@ public DefaultConfiguration(
                 variableNameFunction = variableNameFunctionClass.newInstance();
             } catch (RuntimeException e) {
                 throw e;
+            } catch (ClassNotFoundException e) {
+                throw Throwables.propagate(e);
```

Comment:
Guava intends to deprecate this method, and recommends throwing a RuntimeException directly
https://github.com/google/guava/blob/2bcf189a62275bb23fd92edf4dc72e9ad849a89e/guava/src/com/google/common/base/Throwables.java#L201-L203